test/modules/ingredients/IngredientResolver.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { IngredientResolver } from "../../../src/modules/ingredients/IngredientResolver";
+import { ingredientProviderMock } from "../../__mocks__/IngredientProvider";
+
+let ingredientResolver: IngredientResolver;
+
+beforeAll(() => {
+  ingredientResolver = new IngredientResolver(ingredientProviderMock);
```

Comment:
Schau mal bitte in die Example.spec.ts rein. Das steht wie du GraphQL Queries testest

Reply:
Das wär dann ja aber n Integration Test und kein Unit Test. Machen wir dann beides? Oder doch keine Unit Tests?

Reply:
Beides. Für die API auf jeden falls Integration tests